src/test/java/seedu/recipe/storage/jsonadapters/JsonAdaptedPortionUnitTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.recipe.storage.jsonadapters;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+//import static seedu.recipe.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.recipe.commons.exceptions.IllegalValueException;
+import seedu.recipe.model.recipe.unit.PortionUnit;
+
+public class JsonAdaptedPortionUnitTest {
+    private static final String VALID_UNIT = "grams";
+    private static final String INVALID_UNIT = "123";
```

Comment:
Same wrt this @filbertphang @alson001

Reply:
Realised no way for it throw IllegalValueException as the check occurs earlier.